wisp-time-testing/src/main/kotlin/wisp/time/FakeClock.kt

```diff
@@ -23,6 +24,16 @@ class FakeClock(
     /** Advance the clock by specified [Duration]. */
     fun add(d: Duration) = millis.addAndGet(d.toMillis())
 
+    /**
+     * Advance the clock by the specified [Period].
+     * Note that unlike adding a [Duration] the exact amount that is added to the clock will depend on
+     * its current time and timezone. Not all days, months or years have the same length. See the
+     * documentation for [Period].
+     */
+    fun add(p: Period) = millis.getAndUpdate { millis ->
```

Comment:
@tyiu Is there a reason you used `getAndUpdate` rather than `updateAndGet`? Using `getAndUpdate` means the *old* value is returned from the function which is inconsistent with the other `add` method which uses `addAndGet` and returns the *new* value.

Reply:
I gave it no thought. I just copy pasta'd it from the Misk PR linked in the description. But now that you mention it, I think what you said makes more sense.

Reply:
https://github.com/cashapp/wisp/pull/74